lua/terrortown/lang/zh_hans.lua

```diff
@@ -2153,202 +2153,202 @@ L.help_crosshair_scale_enable = [[
---Role derandomization can be used to make role distribution feel more fair over the course of a session.
---
---In essence, when it is enabled, a player's chance of receiving a role increases while they have not been assigned that role. While this can feel more fair, this also enables metagaming, where a player can guess that another will be traitor-aligned based on the fact that they have not been traitor aligned in several rounds. Do not enable this option if this is undesirable.
---
---There are 4 modes:
---
---mode 0: Disabled - No derandomization is done. This is the default.
---
---mode 1: Base roles only - Derandomization is performed for base roles only. Sub-roles will be selected randomly. These are roles like Innocent and Traitor.
---
---mode 2: Sub-roles only - Derandomization is performed for sub-roles only. Base roles will be selected randomly. Note that sub-roles are only assigned to players which have already been selected for their base role.
---
---mode 3: Base roles AND sub-roles - Derandomization is performed for both base roles and sub-roles.]]
---L.label_roles_derandomize_mode = "Derandomization mode"
---L.label_roles_derandomize_mode_none = "mode 0: Disabled"
---L.label_roles_derandomize_mode_base_only = "mode 1: Base roles only"
---L.label_roles_derandomize_mode_sub_only = "mode 2: Sub-roles only"
---L.label_roles_derandomize_mode_base_and_sub = "mode 3: Base roles AND sub-roles"
-
---L.help_roles_derandomize_min_weight = [[
---Derandomization is performed by making the random player selections during role distribution use a weight associated with each role for each player, and that weight increases by 1 each time the player does not get assigned that role. These weights are not persisted between connections, or across maps.
---
---Each time a player is assigned a role, the corresponding weight is reset to this minimum weight. This weight does not have any absolute meaning; it can only be interpreted with respect to other weights.
---
---For example, given player A with a weight of 1, and player B with a weight of 5, player B is 5 times more likely than player A to be selected. However, if player A had a weight of 4, player B is only 5/4 times more likely to be selected.
---
---The minimum weight, therefore, effectively controls how much each round affects a player's chance at being selected, with higher values causing it to be affected less. The default value of 1 means that each round causes a fairly significant increase in chance, and conversely, that it is extremely unlikely that a player will get the same role twice in a row.
---
---Changes to this value will not take effect until players reconnect or the map changes.]]
---L.label_roles_derandomize_min_weight = "Derandomization minimum weight"
+L.header_roles_derandomize = "角色去随机化"
+
+L.help_roles_derandomize = [[
+角色去随机化可用于使角色分配在会话期间感觉更公平。
+
+本质上，启用后，当玩家没有被分配到某个角色时，他们获得该角色的几率会增加。虽然这可能感觉更公平，但也会引发元游戏，即玩家可以根据其他人多回合没有被分配为叛徒阵营来猜测他们是叛徒。如果这是不可取的，请勿启用此选项。
```

Comment:
```suggestion
系统启用后，当玩家没有被分配到某个角色时，他们之后被选中的概率会更高。虽然这可能感觉更公平，但有被滥用的可能，即可以根据其他人多回合没有被分配为叛徒阵营来猜测他们是叛徒。如果这个后果不理想，请勿启用此选项。
```

Adjusted some phrasing. Metagaming is not a very well known phrase so I elected to avoid the phrase and explain in plainer terms.